### ISSUE TITLE
Add option to keep a device unlocked on sd-devices

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-export#85]
Follow-up for https://github.com/freedomofpress/securedrop-export/pull/39

In the issue description for #39, you'll see:

> We will need to create a followup issue with a new status code to inform the client that the device is already unlocked and no user passphrase is necessary.

The issue for this was created here: https://github.com/freedomofpress/securedrop-export/issues/40

> **Important: ** If the device is unlocked by an external program we will still lock it after a file transfer. This is to err on the side of caution.

No issue was created for this at the time because it was something I thought we would want to support. But once #40 is done, we can revisit this topic and potentially add a new option to keep the device unlocked while continuing to default to locking the device after a transfer completes. If we did this, I think it would make sense to say something more like "Keep device unlocked" rather than "Remember password."

**Comment by creviera** [comment]
> As a journalist who wants to export multiple documents, I'd like to keep the luks-encrypted thumb drive unlocked until I finish all my exports.

This user story will at least partially be addressed by adding support for exporting multiple files from a source.

**Comment by gonzalo-bulnes** [comment]
Some of the new status codes (e.g. `Status.DEVICE_WRITABLE`) introduced in #105 will enable detecting unlocked devices early in the export process if that's something we want to do. That could allow to offer different paths depending on the initial state of the device: like not re-locking a device that wasn't unlocked as part of the export.

No action yet, just flagging that current work will make this easier is we ever want to proceed.
